dashboard/config/routes.rb

```diff
@@ -7,6 +7,9 @@
   # Redirect studio.code.org/courses to code.org/students
   get "/courses", to: redirect(CDO.code_org_url("/students"))
 
+  # Redirect old sign up flow to new sign up flow
+  get "/users/sign_up", to: redirect("/users/new_sign_up/account_type")
```

Comment:
This would override the `registrations#new` controller action (set up by Devise) and leave it abandoned, no? Would it be possible to either:

1. Add the redirect to `registrations#new` (deleting the existing code) using the URL helper via `redirect_to users_new_sign_up_account_type_path`

or

2. Just remove `new_sign_up` in favor of using the simpler `sign_up` and then also updating `registrations#new` to just `redirect_to users_sign_up_account_type_path` (`/users/sign_up/account_type`)?

Reply:
I second doing option 2, but probably in a later PR. Hannah, if you agree, can you make a ticket to capture this?

Reply:
I am trying to avoid hard-coded URL paths in favor of URL helpers since in this case the redirect is internal. Deferring option 2 to the future is certainly fine nonetheless.

Reply:
I captured this work here! Thank you! https://codedotorg.atlassian.net/browse/ACQ-2927